tests/Integration/Execution/MutationExecutor/HasManyTest.php

```diff
@@ -131,7 +188,58 @@ public function testCanCreateHasMany(): void
         ]);
     }
 
-    public function testCanUpdateHasMany(): void
+    public function actionsOverExistingDataProvider()
+    {
+        yield ['Update action' => 'update'];
```

Comment:
I like that pattern, but we do not use it anywhere. Let's keep it consistent.

Reply:
Whoops I didn't find any other provider and I see that it was due to a bad filter in que search. I'm changing it to return with arrays.